client/my-sites/checkout/composite-checkout/hooks/wpcom-store.ts

```diff
@@ -125,14 +121,7 @@ const selectors = {
 	},
 };
 
-export function useWpcomStore(): WpcomCheckoutStore | undefined {
-	// Only register once
-	const registerIsComplete = useRef< boolean >( false );
-	if ( registerIsComplete.current ) {
-		return undefined;
-	}
-	registerIsComplete.current = true;
-
+function createStore() {
```

Comment:
I'd suggest getting rid of the function. If we're going to call it just once, it may just inline its contents.